server/utils/formUtils.ts

```diff
@@ -175,16 +175,20 @@ export function convertKeyValuePairsToSummaryListItems<T extends object>(
   return Object.keys(values).map(key => summaryListItem(titles[key], String(values[key as keyof T])))
 }
 
-export type RenderAs = keyof TextItem | keyof HtmlItem | 'textBlock' | 'date'
+export type RenderAs = keyof TextItem | keyof HtmlItem | 'textBlock' | 'date' | 'shortDate' | 'time'
 
 const renderSummaryValue = (value: string, renderAs: RenderAs) => {
   switch (renderAs) {
     case 'textBlock':
       return { html: `<span class="govuk-summary-list__textblock">${value}</span>` }
     case 'date':
       return { text: (value && DateFormats.isoDateToUIDate(value)) || '' }
+    case 'shortDate':
+      return { text: value && DateFormats.isoDateToUIDate(value, { format: 'short' }) }
```

Comment:
I don't think we need this. Cas1 uses long dates in all the summary lists and short in tables.